### DISCUSSION TITLE
Error while querying : trying to simulate AND condition

### DISCUSSION THREAD

**Discussion by eduardomanuel** [opensource9ja/danfojs#466] [Q&A]
Hi i read on another post that multiple conditions are not supported while using the df.query() function.   So, i am trying to simulate an AND condition by querying a dataframe that is the result of a query.  Please see image below and let me know if i am doing something wrong?
![error-df-query](https://user-images.githubusercontent.com/13238743/176317179-cec65e23-2ea7-4256-b656-457bfaed1f12.PNG)

**Comment by jmajety-dev** [comment]
I also have a same issue please let me know, What can be done ?
